livekit-android-sdk/src/main/java/io/livekit/android/room/participant/LocalParticipant.kt

```diff
@@ -267,9 +272,23 @@ internal constructor(
         options: VideoTrackPublishOptions = VideoTrackPublishOptions(null, videoTrackPublishDefaults),
         publishListener: PublishListener? = null,
     ) {
+        val isSVC = isSVCCodec(options.videoCodec)
+
+        @Suppress("NAME_SHADOWING") var options = options
+        if (isSVC) {
+            dynacast = true
+
+            // Ensure backup codec and scalability for svc codecs.
+            if (options.backupCodec == null) {
+                options = options.copy(backupCodec = BackupVideoCodec())
```

Comment:
I guess this would enable multicodec automatically if null?

Reply:
I do like that default, but feels like it would be a bit inconsistent if there isn't a way to turn it off.

JS currently allows users to set it to `true` to enable with defaults: https://github.com/livekit/client-sdk-js/pull/923